pages/dkp/kommander/2.1/install/configuration/index.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: layout.pug
+navigationTitle: Configuration
+title: Configuration
+menuWeight: 30
+excerpt: How to configure a Kommander installation
+beta: false
+enterprise: false
+---
+
+Kommander can be configured during the initial installation and also post-installation using the Kommander CLI.
+
+## Initializing a configuration file
+
+To get started with configuring Kommander, you can run the following command to initialize a default configuration file:
+
+```bash
+kommander install --init > kommander.yaml
+```
+
+## Configuring applications
+
+Once you have a default configuration file, each `app` can be configured either inline **or** by referencing another `yaml` file. The configuration values for each `app` correspond to the Helm Chart values for the application.
+
+### Inline configuration (using values)
+
+In this example, we configure the `centralized-grafana` application with resource limits by defining the Helm Chart values in the Kommander configuration file.
```

Comment:
```suggestion
In this example, you configure the `centralized-grafana` application with resource limits by defining the Helm Chart values in the Kommander configuration file.
```

Reply:
resolved in a2389daa9fd58b84770bb55c6ee73e7b4e829e54